service/src/main/kotlin/fi/espoo/evaka/shared/dev/DevApi.kt

```diff
@@ -388,6 +389,12 @@ DELETE FROM attachment USING ApplicationsDeleted WHERE application_id = Applicat
         return ResponseEntity.ok().build()
     }
 
+    @DeleteMapping("/employee/all")
+    fun deleteEmployees(db: Database): ResponseEntity<Unit> {
+        db.transaction { it.handle.deleteEmployees() }
```

Comment:
Btw are mobile devices cascaded or do we need to first delete them to avoid errors? This could also fail with any other references, like absences/decisions created by the employee. So now I'm starting to think this whole approach of deleting all employees does not work. Either we need to surgically delete correct things in correct order or go for the full db nuke. This level of partial nuking is likely to just cause more issues.